README.md

```diff
@@ -89,11 +89,11 @@ $ export FLASK_CONFIGURATION=test
 . So that tests are not dependent on any env variable. By default it is dependent on
 local postgresql instance.
 
-All tests are in tests directory. We use nose for testing
+All tests are in tests directory. We use pytest for testing
 
-To run all tests plz use ```nosetests tests``` from the base directory.
-If want to run specific module use i.e. ```nosetests tests/test_basics.py```
+To run all tests plz use ```pytest tests``` from the base directory.
```

Comment:
Would be better to have a command on a separate line for easy copy and paste e.g.


To run all tests please run this in the base directory:

```
pytest tests
```